player/player.tscn

```diff
@@ -22,6 +22,7 @@ input_pickable = false
 shape = SubResource("CircleShape2D_vfvd7")
 
 [node name="Sprite2D" type="Sprite2D" parent="."]
+visible = false
```

Comment:
Does this need to be removed instead?

Reply:
i just changed the visibility for the player sprite while making the other changes and forgot. can remove the sprite node altogether i think.